x-pack/plugins/security_solution/common/detection_engine/rule_management/api/rules/info/response_schema.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as t from 'io-ts';
+import { PositiveInteger } from '@kbn/securitysolution-io-ts-types';
+
+export type RulesInfoResponse = t.TypeOf<typeof RulesInfoResponse>;
+export const RulesInfoResponse = t.exact(
+  t.type({
+    rules_custom_count: PositiveInteger,
+    rules_prebuilt_installed_count: PositiveInteger,
+    tags: t.array(t.string),
+  })
```

Comment:
I would propose using a nested response structure to allow further extensibility of that endpoint. For example:

```ts
{
  rules_summary: {
    custom_count: PositiveInteger,
    prebuilt_installed_count: PositiveInteger,
  },
  aggregated_fields: {
    tags: t.array(t.string),
  },
}
```

Reply:
Good point. I though about splitting into categories but implemented as a plain object. It makes sense to split as we plan to extend the response.